cmd/config.go

```diff
@@ -191,6 +191,8 @@ type Config struct {
 	AllowedSigningAlgos *AllowedSigningAlgos
 
 	SubscriberAgreementURL string
+
+	Features map[string]bool
```

Comment:
Per the config plan, this should be a type that is embedded into the config struct of each component that uses it.

Also, why string rather than a struct? The latter is nice because you get checking for typos, and you can be sure that once a flag is gone, it's gone. I guess the string types are a way of skirting circular dependencies between config and the various components?